examples/components/paging/splitted_usage.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../ng2-material/core/style/variables";
+@import "../../../ng2-material/core/style/default-theme";
+@import "../../../ng2-material/components/whiteframe/whiteframe";
+
+paging-splitted-usage {
```

Comment:
`split-usage` is fine. The past tense `splitted` isn't really correct English anyway.